src/infra/metrics/metrics_definitions.rs

```diff
@@ -8,10 +8,10 @@ metrics! {
     gauge rpc_requests_active{client, method},
 
     "Number of JSON-RPC requests that started."
-    counter rpc_requests_started{client, method, function},
+    counter rpc_requests_started{client, method, function, from, to},
 
     "Number of JSON-RPC requests that finished."
-    histogram_duration rpc_requests_finished{client, method, function, result, result_code, success},
+    histogram_duration rpc_requests_finished{client, method, function, from, to, result, result_code, success},
```

Comment:
Won't these new labels increase the cardinality of these metrics by a lot? It seems `from` and `to` will be label values that are very distinct over time.

Reply:
How unique are these values?

Reply:
I agree with Aranha here, this is a useful piece of info but unfortunately Grafana doesn't allow us to increase cardinality like this, this would break our metrics as it indefinitely increases the memory footprint.